apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Generated by `createschema miniapp.B2BAppAccessRightSet 'app:Relationship:B2BApp:CASCADE;'`
+ */
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2BAppAccessRightSet')
+
+
+const B2BAppAccessRightSet = new GQLListSchema('B2BAppAccessRightSet', {
+    schemaDoc: 'A set of access rights for a service user integrated with B2BApp.',
```

Comment:
Please provide more description to all miniapps models, since they're actively used by our partners, so it important for this descriptions to be as clear as possible